posthog/api/cohort.py

```diff
@@ -1198,13 +1198,6 @@ def perform_create(self, serializer):
         serializer.save()
         instance = cast(Cohort, serializer.instance)
 
-        # Although there are no changes when creating a Cohort, we synthesize one here because
-        # it is helpful to show the list of people in the cohort when looking at the activity log.
```

Comment:
It has been helpful when people report issues with static cohorts to have the changes in the activity log.

Couple ideas:
1. Continue to show the changes if the changes are under some reasonable threshold
2. Or only show the number of people added/removed.

I don't think this is a blocker though.

Reply:
Ahh, I went back and forth on this myself but I hadn't considered the usefulness in troubleshooting. I've changed it to report the number of people added.